lib/http2/connection.c

```diff
@@ -278,7 +277,7 @@ int close_connection(h2o_http2_conn_t *conn)
     return 0;
 }
 
-void send_stream_error(h2o_http2_conn_t *conn, uint32_t stream_id, int errnum)
+void h2o_http2_stream_send_error(h2o_http2_conn_t *conn, uint32_t stream_id, int errnum)
```

Comment:
With the introduction of the tri-state value, I think we can leave the function as `static`.